docs/conf.py

```diff
@@ -0,0 +1,52 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
We already have a configuration [in the repo](https://github.com/DesignSafe-CI/portal/tree/master/.docs/source/conf.py)